include/FtyPolicies.hpp

```diff
@@ -6,26 +10,28 @@
 
 namespace fty {
 struct AsLowercase {
-  std::string apply(const std::string &String) {
+  static auto apply(const std::string& String) -> std::string {
```

Comment:
Why replace the return type with auto and then give it a trailing return type here?